gnovm/stdlibs/math/overflow/overflow.gno

```diff
@@ -2,8 +2,10 @@
 // NOTE: there was a bug with the original Quotient* functions, and
 // testing method. These have been fixed here, and tests ported to
 // tests/files/maths_int*.go respectively.
-// TODO: make PR upstream.
-package maths
+// Note: copied over from p/demo/maths & deprecated over there.
```

Comment:
```suggestion
// Note: moved from p/demo/maths.
```

(they're removed, not deprecated)

Reply:
[9574b6c](https://github.com/gnolang/gno/pull/1698/commits/9574b6cc9d950a5809106bf5ab2cc2b930937a87)